content/data-reporting/changelog/index.mdx

```diff
@@ -11,7 +11,21 @@ This document is the record of all the changes in [**the Reports API**](/data-re
 
 ## [v3.4] - Developer preview
 
-No changes yet.
+- POST method becomes primary and recommended way of sending requests. GET method is still supported, however due to limitations of query string length might fail for more complex requests.
+- Request format has a new format, the filters are nested within the `filters` field now.
+- To provide more consistent experience across Platform APIs, filters were reworked and aligned with those used in the **List Archives** ([Web](/messaging/agent-chat-api/v3.4/#list-archives). All reports:
```

Comment:
Honest question:  is `PlatformAPIs` how we refer to our APIs in the docs?

Reply:
Yes